pages/design/Design Guidelines - Designers.md

```diff
@@ -0,0 +1,221 @@
+
+#### Last Step
+After you made all your changes, make sure to select **Cover / Master** Frame, right clicking on it and then selecting **Set as Thumbnail**. The changes will apply after you restart Figma app.
+
+
+## Components and Variants
+Working with components and variants, we make sure that the changes that need to be done are made in the most efficient and the least time consuming method.
+In the Starter, as a global components and variants you will find:
+- Buttons
+- Checkbox
+- Radio Buttons
+- Dropdowns
+- Icons
+- Input Fields
+- Pills/Badges
+- Alerts
+- Navigation Bars
+- Footers
+- Modals
+- Paginaters
+
+#### Buttons
+The button component is made out these variants: 
+- 3 Sizes
+- Button with Icon variants
+- Text Only Variants
+- All components made with State instances
+
+Tweaking the button is as easy as Importing it from Components panel, and selecting the properties from the Instance panel.
+
+#### Checkbox
+The checkbox component is made out of 3 instances:
+- Checked
+- Unchecked
+- Multiple
+- All components made with State instances
+
+#### Radio Button
+The Radio Button component is made out of 2 instances:
+- Checked
+- Unchecked
+- All components made with State instances
+
+#### Dropdowns
+Added in the Starter, you can find Components for Dropdowns.
+You can find Two types of it:
+- Closed
+- Open
+- All components made with State instances
+
+#### Icons
+The Icons are made from multiple components. The structure goes by creating a rectangular Frame with 24px width and height, and then inserting the icon inside the Frame. The Frame should have a name by: Icon/<iconname>. The renaming it's very important because of the way how the Figma component naming works. The Icon/ is made as a nested folder and if you keep the name the same, changing them it's very easy!
+
+#### Input Fields
+Input Fields components are made by multiple instances:
+- Inside Text (with and without icons)
+- Outside Text (with and without icons)
+- Optional Texts
+- Help Texts
+- Leading and Trailing Icons
+- All components made with State instances
+
+#### Pills (Badges)
+You can find two components for Pills:
+- Dots
+- X Icons
+- Color Variants
+
+#### Pagination
+Pagination set is presented by 4 Components:
+- Hover
+- Active
+- Selected
+- Disabled
+
+## Text Styles
+On the Project Starter, we have implemented the default Text Styles for TailwindCSS. You can find ~150 Font Styles. Globally divided into two font styles, Named on Font 1, Font 2; The fonts are divided onto **General** and **Italic** global folders. Changing text style is as easy as selecting the Style Icon, and on search input field typing for e.x: `Font 1 SM Regular`. This will select the `Font 1, SM/14px size, regular weight.` By needs, we switch the typography structure by 'Heading' and 'Paragraph' Styles. We generate them with increment by the needs. Importing them via the `Styler` app is very useful, also adapting the properties via `Batch Styler`.
+
+## Color Styles
```

Comment:
Why you should add default colors provided by tailwindcss

Reply:
Because they are not automatically added in Figma. We use a plugin to input colors as styles. Updating them it's much easier for the team